pkgs/development/python-modules/nsenter/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchPypi, contextlib2, argparse, flake8, pathlib, pythonOlder, pythonAtLeast }:
+
+buildPythonPackage rec {
+  pname = "nsenter";
+  version = "0.2";
```

Comment:
this hasn't seen an update in 4 years, it probably shouldn't go in nixpkgs

Reply:
There have been new changes to the repository as recently as 16 days ago and there is no hint that it is unmaintained. If you want I can also package the latest master revision, but there haven't been significant changes, mostly because it still works fine as it is.